AmplifyPlugins/Core/AWSPluginsCore/ServiceConfiguration/AmplifyAWSServiceConfiguration.swift

```diff
@@ -0,0 +1,37 @@
+//
+// Copyright 2018-2019 Amazon.com,
+// Inc. or its affiliates. All Rights Reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+import Foundation
+import AWSCore
+
+public class AmplifyAWSServiceConfiguration: AWSServiceConfiguration {
+    static let baseUserAgent: String = {
```

Comment:
[nit] either in test comment or here, i think it would be nice to have an example of what this actually generates, ie.
```
/// Returns a string representing the user agent like `Amplify-iOS/1.0.0 OS X/13.0 en-US`
```

Reply:
I noticed that you commented on this before, and I thought long and hard about putting a comment here.  I decided that putting a comment here was more work than it's worth because the code is obvious and (like all comments) require maintenance.  That being said, if you feel strongly about this, i can add a comment.

Reply:
As noted above, since you've made this a static property, your call sites should just use `AmplifyAWSServiceConfiguration.baseUserAgent` not `AmplifyAWSServiceConfiguration.baseUserAgent()`, unless I'm missing something.